docs/design/lexical_conventions/symbolic_tokens.md

```diff
@@ -0,0 +1,106 @@
+| `*`             | Indirection, multiplication, and forming pointer types                                                     |
+| `/`             | Division                                                                                                   |
+| `%`             | Modulus                                                                                                    |
+| `=`             | Assignment                                                                                                 |
+| `^`             | Complementing and Bitwise XOR                                                                              |
+| `&`             | Address-of and Bitwise AND                                                                                 |
+| `\|`            | Bitwise OR                                                                                                 |
+| `<<`            | Arithmetic and Logical Left-shift                                                                          |
+| `>>`            | Arithmetic and Logical Right-shift                                                                         |
+| `==`            | Equality or equal to                                                                                       |
+| `!=`            | Inequality or not equal to                                                                                 |
+| `>`             | Greater than                                                                                               |
+| `>=`            | Greater than or equal to                                                                                   |
+| `<`             | Less than                                                                                                  |
+| `<=`            | Less than or equal to                                                                                      |
+| `->`            | Return type and indirect member access                                                                     |
+| `=>`            | Match syntax                                                                                               |
+| `[`             | Subscript and deduced parameter lists                                                                      |
+| `]`             | Subscript and deduced parameter lists                                                                      |
+| `(`             | Function call, function declaration and tuple literals                                                     |
+| `)`             | Function call, function declaration and tuple literals                                                     |
+| `{`             | Struct literals, blocks of control flow statements and the bodies of definitions (classes, functions, etc) |
+| `}`             | Struct literals, blocks of control flow statements and the bodies of definitions (classes, functions, etc) |
```

Comment:
This is the kind of thing I was suggesting in my [previous comment](https://github.com/carbon-language/carbon-lang/pull/2657#discussion_r1156333901):

```suggestion
| `[` and `]`     | Subscript and deduced parameter lists                                                                      |
| `(` and `)`     | Function call, function declaration and tuple literals                                                     |
| `{` and `}`     | Struct literals, blocks of control flow statements and the bodies of definitions (classes, functions, etc) |
```

Reply:
Ok, I will add these!